Vostok.Metrics.System/Host/NativeHostMetricsCollector_Linux.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using Vostok.Metrics.System.Helpers;
+
+// ReSharper disable PossibleInvalidOperationException
+
+namespace Vostok.Metrics.System.Host
+{
+    internal class NativeHostMetricsCollector_Linux : IDisposable
+    {
+        private readonly ReusableFileReader systemStatReader = new ReusableFileReader("/proc/stat");
+        private readonly ReusableFileReader memoryReader = new ReusableFileReader("/proc/meminfo");
+        private readonly HostCpuUtilizationCollector cpuCollector = new HostCpuUtilizationCollector();
+
+        public void Dispose()
+        {
+            systemStatReader.Dispose();
+            memoryReader.Dispose();
+        }
+
+        //https://supportcenter.checkpoint.com/supportcenter/portal?eventSubmit_doGoviewsolutiondetails=&solutionid=sk65143
```

Comment:
Probably shouldn't quote such a questionable source. Use the well-known manual for proc filesystem: https://man7.org/linux/man-pages/man5/proc.5.html